linux/Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM alpine:edge
+
+# TODO: change to container registry
+COPY --from=clangbuiltlinux/llvm-project:latest llvm-project/llvm/build/bin /usr/local/bin
+COPY --from=clangbuiltlinux/llvm-project:latest /usr/local/lib /usr/local/lib
+COPY --from=clangbuiltlinux/llvm-project:latest /usr/local/include /usr/local/include
+RUN cd /usr/lib/ && \
+  for library in libc++abi.so.1 libc++.a libc++abi.a libc++.so.1 libunwind.so.1 libunwind.a; \
+    do ln -s "/usr/local/lib/x86_64-alpine-linux-musl/${library}" . ; \
```

Comment:
I think having the `do` on the previous line is more standard (this is in a lot of places so I'll just comment on it here).

Reply:
yeah, idk I chose this line break, will fix.

Reply:
done. PTAL